opm/output/eclipse/Summary.cpp

```diff
@@ -801,6 +819,13 @@ void Summary::add_timestep( int report_step,
     this->prev_time_elapsed = secs_elapsed;
 }
 
+void Summary::set_initial( const data::Solution& sol ) {
```

Comment:
I don't know if we will need more of these initial properties; but at least for the second case I think we should use a map-like structure to internalize these initial values.